src/components/Form.js

```diff
@@ -29,35 +29,37 @@ const Form = () => {
 
   return (
     <form className="form">
-      <input
-        type="text"
-        name="title"
-        placeholder="Author"
-        className="title"
-        onChange={handleChange}
-      />
-      <input
-        type="text"
-        name="author"
-        placeholder="Book"
-        className="author"
-        onChange={handleChange}
-      />
-      <select value={bookInfo.category} onChange={handleChange} name="category">
-        <option value="History">History</option>
-        <option value="Romance">Romance</option>
-        <option value="Mystery">Mystery</option>
-        <option value="Science">Science</option>
-        <option value="Technology">Technology</option>
-      </select>
-
-      <input
-        type="submit"
-        value="Submit"
-        className="submit"
-        onClick={handleClick}
-      />
+      <span>ADD NEW BOOK</span>
+      <div className="inner--form">
```

Comment:
- [ ] Your design does not fit the screen. Your page is scrolling horizontally. I recommend fixing the width of your layouts to improve your UI design and the user experience. Ensure that there is no horizontal bar on your page.

The Form:
<img width="946" alt="image" src="https://user-images.githubusercontent.com/77190203/185745732-ceadec91-41a3-4c92-b15e-aca94eff07b4.png">

The header:
<img width="947" alt="image" src="https://user-images.githubusercontent.com/77190203/185745774-560675d2-d41c-4159-bb04-1cb16ccbfc81.png">